domenetjenester/arbeidsforhold-domene/src/main/java/no/nav/ung/sak/domene/iay/modell/InntektArbeidYtelseAggregat.java

```diff
@@ -1,27 +1,19 @@
 package no.nav.ung.sak.domene.iay.modell;
 
-import java.time.LocalDateTime;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.LinkedHashSet;
-import java.util.Objects;
-import java.util.Set;
-import java.util.UUID;
-import java.util.stream.Collectors;
-
 import no.nav.ung.sak.behandlingslager.diff.ChangeTracked;
 import no.nav.ung.sak.behandlingslager.diff.DiffIgnore;
 
+import java.time.LocalDateTime;
+import java.util.*;
+import java.util.stream.Collectors;
+
 public class InntektArbeidYtelseAggregat {
 
     private UUID uuid;
 
     @ChangeTracked
     private Set<AktørInntekt> aktørInntekt = new LinkedHashSet<>();
 
-    @ChangeTracked
-    private Set<AktørYtelse> aktørYtelse = new LinkedHashSet<>();
```

Comment:
Hva er aktørYtelse?
Er det anderledes enn info om NAV ytelsene (SYK og OMS) vi skal støtte?

Reply:
Aktørytelse er data om vedtaksperiodene til dei ulike ytelsene. Det er bestemt at vi skal forholde oss til kontantprinsippet og dermed er det måneden pengene utbetales i som har noko å seie for om ytelsen skal avkorte/redusere utbetaling av ungdomsytelsen eller ikkje. Dermed trenger vi ikkje, slik det ser ut no, å hente inn informasjon om vedtaka til ytelsene, men det er nok med data fra a-ordningen. Dette kan endres på sikt,  og derfor har eg splittet ut dette i ein eigen commit slik at det er enklere å ta inn igjen om det skulle vere nødvendig.

Reply:
Den er grei. Takk for forklaringen 👍